src/backend/env/user.rs

```diff
@@ -611,6 +611,22 @@ impl User {
             Ok(())
         })
     }
+
+    /** Protect against invite phishing */
+    pub fn validate_send_credits(&self, state: &State) -> Result<(), String> {
```

Comment:
We should a todo here: since credits burned get reset every week, some users will never be able to transfer credits until the invite is alive. But we can handle this at some point later.

Reply:
Quite a surprise we don't keep the number. We could add a metric `credits_burned_total` which wouldn't reset?